typings/index.d.ts

```diff
@@ -1305,6 +1305,10 @@ declare module 'discord.js' {
 		public status: Status;
 		public readonly ping: number;
 
+		public broadcast(packet: object): void;
```

Comment:
This should be removed, as it's already mentioned in the private functions part of the class